wds-disable-pingbacks.php

```diff
@@ -35,11 +41,47 @@ function wds_disable_pingbacks( $methods ) {
  */
 function wds_disable_pingbacks_options( $screen ) {
 	if ( 'options-discussion' === $screen->id ) {
-		wp_register_script( 'wds-disable-pingbacks', plugins_url( 'wds-disable-pingbacks.js', __FILE__ ), 'jquery' );
-		$translations = array(
+		wp_register_script( 'wds-disable-pingbacks', plugins_url( 'wds-disable-pingbacks.js', __FILE__ ), 'jquery' ); // @codingStandardsIgnoreLine: Leave in header.
+
+		$translations = [
```

Comment:
Unless the array is going to be used more than once, you can probably just drop it in `wp_localize_script` rather than set it to a variable (unless this is needed for i18n for some reason).

Reply:
done.